mise.toml

```
@@ -1,2 +1,3 @@
 [tools]
 deno = "2.4.3"
+pnpm = "10.12.4"
```

I think it should be:

```suggestion
"npm:pnpm" = "10.12.4"
```

---

I was unfamiliar for mise and it seems better to use full name of the tool. Thanks!

https://mise.jdx.dev/registry.html?filter=pnpm#tools